director/director_api.go

```diff
@@ -118,3 +145,17 @@ func ConfigTTLCache(ctx context.Context, egrp *errgroup.Group) {
 		return nil
 	})
 }
+
+// Populate internal filteredServers map by Director.FilteredServers
+func ConfigFilterdServers() {
+	filteredServersMutex.Lock()
+	defer filteredServersMutex.Unlock()
+
+	if len(param.Director_FilteredServers.GetStringSlice()) == 0 {
```

Comment:
If it's helpful, I recently added an `IsSet` interface for most of our internal param types:

https://github.com/PelicanPlatform/pelican/blob/cae5ec86cd8dc4d6564035d3880a85f6b854418f/param/parameters.go#L77-L79